### PR TITLE
fix broken URL in Lessons learned from troubleshooting ROBOT

### LINKED ISSUE #232: Weird-looking obook page is the evil twin of a real obook page
https://oboacademy.github.io/obook/lesson/ontology_pipelines/ looks weird
The correct page is https://oboacademy.github.io/obook/lesson/ontology-pipelines/ but I got to the underscore mutant version from https://oboacademy.github.io/obook/reference/troublehooting-robot/  (which I notice has a typo in the URL, not that that really matters), which I got to by searching for ROBOT in the obook search.

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
- remove second prerequisite because it links to the same URL
close #232

### CODE REVIEW

File: docs/lesson/ontology-term-use.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ The figure below by [Chris Mungall](https://bids.berkeley.edu/people/chris-munga
 <a name="request"></a> 
 ## 7. Make term requests to existing ontologies
 
-See separate lesson on [Making term requests to existing ontologies](../how-to/term-request.md).
+See separate lesson on [Making term requests to existing ontologies](https://oboacademy.github.io/obook/howto/term-request/).
```
Comment (matentzn) [inline docs/lesson/ontology-term-use.md:124]: Always use relative links when linking to other obook pages `../howto/term-request.md`

File: docs/lesson/semantic-database-fundamentals.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 These materials are under construction and incomplete.
 
 ## Prerequisites
-- Review tutorial on [Application Ontology Development](/redesign/05_applicationOntologyDevelopment.md)
+- TBD
```
Comment (matentzn) [inline docs/lesson/semantic-database-fundamentals.md:7]: https://oboacademy.github.io/obook/tutorial/project-ontology-development/

(remember: `../tutorial/project-ontology-development.md`)

File: docs/reference/ontology-landscape.md [hunk 3]
```diff
@@ -0,0 +1,561 @@
+## The Landscape of Disease and Phenotype Ontologies
```
Comment (matentzn) [inline docs/reference/ontology-landscape.md:1]: rename file or do you intende this to be about all kinds of ontologies? maybe better: `medical-ontology-landscape.md`?

File: docs/reference/troublehooting-robot.md [hunk 4]
```diff
@@ -1,8 +1,7 @@
 # Lessons learned from troubleshooting ROBOT
 
 ## Prerequisites
-- Review tutorial on [Ontology pipelines with ROBOT and SPARQL](https://oboacademy.github.io/obook/lesson/ontology_pipelines/)
-- Review tutorial on [Ontology Pipelines with ROBOT 2](https://oboacademy.github.io/obook/lesson/ontology_pipelines/)
+- Review tutorial on [Ontology pipelines with ROBOT and SPARQL](https://oboacademy.github.io/obook/lesson/ontology-pipelines/)
```
Comment (matentzn) [inline docs/reference/troublehooting-robot.md:4]: ../lesson/ontology-pipelines.md

File: docs/tutorial/github-fundamentals.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ## Introduction to GitHub
-##### [Back to Getting Started](https://github.com/jamesaoverton/obook/tree/master/00-gettingStarted)
+##### [Back to Getting Started](https://oboacademy.github.io/obook/getting-started/)
```
Comment (matentzn) [inline docs/tutorial/github-fundamentals.md:2]: `../getting-started.md`

**Review by matentzn — CHANGES_REQUESTED** [review]

File: docs/reference/troublehooting-robot.md (hunk 4 above)
Comment (nicolevasilevsky) [inline docs/reference/troublehooting-robot.md:4]:
```suggestion
- Review tutorial on [Ontology pipelines with ROBOT and SPARQL](../lesson/ontology-pipelines.md)
```

File: docs/lesson/developing-application-ontologies.md [hunk 6]
```diff
@@ -4,7 +4,7 @@
 These materials are under construction and incomplete.
 
 ## Prerequisites
-- Review tutorials on [Ontology Theory](/redesign/03_ontologyTheory.md) and [Ontology Contribution](/redesign/02_ontologyContribution.md)
+- Review tutorial on [Ontology Contribution](lesson/contributing-to-obo-ontologies.md)
```
Comment (nicolevasilevsky) [inline docs/lesson/developing-application-ontologies.md:7]:
```suggestion
- Review tutorial on [Ontology Contribution](..lesson/contributing-to-obo-ontologies.md)
```

File: docs/lesson/developing-application-ontologies.md [hunk 7]
```diff
@@ -4,7 +4,7 @@
 These materials are under construction and incomplete.
 
 ## Prerequisites
-- Review tutorials on [Ontology Theory](/redesign/03_ontologyTheory.md) and [Ontology Contribution](/redesign/02_ontologyContribution.md)
+- Review tutorial on [Ontology Contribution](..lesson/contributing-to-obo-ontologies.md)
```
Comment (nicolevasilevsky) [inline docs/lesson/developing-application-ontologies.md:7]:
```suggestion
- Review tutorial on [Ontology Contribution](../lesson/contributing-to-obo-ontologies.md)
```

File: docs/lesson/ontology-term-use.md (hunk 1 above)
Comment (nicolevasilevsky) [inline docs/lesson/ontology-term-use.md:124]:
```suggestion
See separate lesson on [Making term requests to existing ontologies](../howto/term-request/).

```

File: docs/lesson/ontology-term-use.md [hunk 8]
```diff
@@ -121,7 +121,8 @@ The figure below by [Chris Mungall](https://bids.berkeley.edu/people/chris-munga
 <a name="request"></a> 
 ## 7. Make term requests to existing ontologies
 
-See separate lesson on [Making term requests to existing ontologies](../how-to/term-request.md).
+See separate lesson on [Making term requests to existing ontologies](../howto/term-request/).
```
Comment (nicolevasilevsky) [inline docs/lesson/ontology-term-use.md:124]:
```suggestion
See separate lesson on [Making term requests to existing ontologies](../howto/term-request.md).

```

File: docs/reference/ontology-landscape.md (hunk 3 above)
Comment (nicolevasilevsky) [reply docs/reference/ontology-landscape.md:1]: I don't intend to make this about all kinds of ontologies but someone else could. :) I'll rename to medical-ontology-landscape

File: docs/lesson/using-disease-and-phenotype-ontologies.md [hunk 9]
```diff
@@ -40,11 +40,11 @@ Different ontologies are build for different purposes and were created for vario
...
+Different disease ontologies may be built for different purposes; for example, ontologies like Mondo and DO are intended to be used for classifying data, and downstream computational analyses. Some terminologies are used for indexing purposes, such as International classification of Diseases (ICD). ICD-11 is intended for indexing medical encounters for the purposes of billing and coding. Some of the disease ontologies listed on the [landscape](..reference/medical-ontology-landscape.md) contain terms that define diseases, such as [Ontology for General Medical Sciences (OGMS)](http://obofoundry.org/ontology/ogms.html) are upper level ontologies and are intended for integration with other ontologies.
```
Comment (nicolevasilevsky) [inline docs/lesson/using-disease-and-phenotype-ontologies.md:47]: note to self, this link is not working

File: docs/lesson/using-disease-and-phenotype-ontologies.md [hunk 10]
```diff
@@ -31,7 +31,7 @@ These materials are under construction and incomplete.
 <a name="major"></a> 
 ## Major disease and phenotype ontologies that are available
 
-A landscape analysis of major disease and phenotype ontologies that are currently available is [here](https://github.com/OBOAcademy/obook/blob/master/archive_original_cpath_course/09-DiseasesAndPhenotypes/ontologylandscape.md).
+A landscape analysis of major disease and phenotype ontologies that are currently available is [here](..reference/medical-ontology-landscape.md) (also available in Zenodo [here](https://zenodo.org/record/6299898#.YvM_kezMJ6od)).
```
Comment (nicolevasilevsky) [inline docs/lesson/using-disease-and-phenotype-ontologies.md:34]: note to self, this link is not working

File: docs/tutorial/github-fundamentals.md (hunk 5 above)
Comment (nicolevasilevsky) [inline docs/tutorial/github-fundamentals.md:2]:
```suggestion
##### [Back to Getting Started](../getting-started/)
```

File: docs/lesson/ontology-term-use.md (hunk 1 above)
Comment (nicolevasilevsky) [reply docs/lesson/ontology-term-use.md]: will do, thanks!

File: docs/lesson/using-disease-and-phenotype-ontologies.md (hunk 10 above)
Comment (nicolevasilevsky) [inline docs/lesson/using-disease-and-phenotype-ontologies.md:34]:
```suggestion
A landscape analysis of major disease and phenotype ontologies that are currently available is [here](../reference/medical-ontology-landscape.md) (also available in Zenodo [here](https://zenodo.org/record/6299898#.YvM_kezMJ6od)).
```

File: docs/lesson/using-disease-and-phenotype-ontologies.md (hunk 9 above)
Comment (nicolevasilevsky) [inline docs/lesson/using-disease-and-phenotype-ontologies.md:47]:
```suggestion
Different disease ontologies may be built for different purposes; for example, ontologies like Mondo and DO are intended to be used for classifying data, and downstream computational analyses. Some terminologies are used for indexing purposes, such as International classification of Diseases (ICD). ICD-11 is intended for indexing medical encounters for the purposes of billing and coding. Some of the disease ontologies listed on the [landscape](../reference/medical-ontology-landscape.md) contain terms that define diseases, such as [Ontology for General Medical Sciences (OGMS)](http://obofoundry.org/ontology/ogms.html) are upper-level ontologies and are intended for integration with other ontologies. 
```

Comment (nicolevasilevsky) [reply docs/lesson/using-disease-and-phenotype-ontologies.md]: fixed

File: docs/lesson/using-disease-and-phenotype-ontologies.md (hunk 10 above)
Comment (nicolevasilevsky) [reply docs/lesson/using-disease-and-phenotype-ontologies.md]: fixed

**Pull request merged**